pkg/dns/azure/dns.go

```diff
@@ -38,6 +39,8 @@ type Config struct {
 	ARMEndpoint string
 	// InfraID is the generated ID that is used to identify cloud resources created by the installer.
 	InfraID string
+	// Tags defined by user which will applied on new resources created by operator
```

Comment:
```suggestion
	// Tags is a map of user-defined tags which should be applied to new resources created by the operator.
```

Reply:
incorporated comment along with the other comments.